libpod/container_inspect.go

```diff
@@ -894,3 +870,27 @@ func (c *Container) GetDevices(priv bool, ctrSpec spec.Spec, deviceNodes map[str
 	}
 	return devices, nil
 }
+
+func BlkioDeviceThrottle(deviceNodes map[string]string, devs []spec.LinuxThrottleDevice) ([]define.InspectBlkioThrottleDevice, error) {
```

Comment:
Does this need to be public? If so, can you move it to a utility package (pkg/utils probably best)?

Reply:
made it private, it is only used w/in libpod, thanks for catching that.